Sources/SnapshotTesting/Snapshotting/NSBezierPath.swift

```diff
@@ -38,6 +40,7 @@ extension Snapshotting where Value == NSBezierPath, Format == String {
   /// A snapshot strategy for comparing bezier paths based on pixel equality.
   ///
   /// - Parameter numberFormatter: The number formatter used for formatting points.
+  @available(macOS 11.0, *)
```

Comment:
These two availability attributes are the only ones truly required to fix the build with Xcode 15.